content/stack/install/add-module-to-cluster.md

```diff
@@ -24,7 +24,10 @@ To use other modules or upgrade an existing module to a more recent version, you
 
 To install or upgrade a module on a [Redis Enterprise]({{<relref "/rs">}}) cluster, you need a module package.
 
-- For Redis Enterprise modules, download packages from the [Redis download center](https://redislabs.com/download-center/modules/).
+- For the latest Redis Enterprise modules, download packages from the [Redis download center](https://redislabs.com/download-center/modules/).
+
+- For earlier versions of Redis Enterprise modules that are no longer available from the Redis download center, [contact support](https://redis.com/company/support/).
+
 - For custom-packaged modules, either download a [custom-packaged module](https://redislabs.com/community/redis-modules-hub/) from the developer or [package the module yourself]({{<relref "/stack/install/packaging-modules">}}).
```

Comment:
Do we support this or want people to do this?  I know for sure we don't support people creating and using their own modules.

Reply:
Maybe its necessary in certain cases when supporting a preview.  Not sure, I feel like we should confirm this.  Maybe I am off.

Reply:
@K-Jo @LiorKogan What are your thoughts on this? Should I remove the following statement from the Enterprise docs?

> For custom-packaged modules, either download a [custom-packaged module](https://redislabs.com/community/redis-modules-hub/) from the developer or [package the module yourself](https://docs.redis.com/latest/stack/install/packaging-modules/).